### PR TITLE
Add Sidekiq Admin UI Request yaml file

### PR DESCRIPTION
**Pull request by ryan-mcneil**
Added a form comparable to those that we use for the AWS and Terminal Access Requests, among many others.

You'll notice I've put the responsibility into the hands of the requester to:
* Indicate which Roster they're on
* Indicate which "method" they'd like us to confirm their EQIP status.

It also indicates they should not continue creating the issue until they've done the required steps, so that we don't have unfinished tickets that hang for weeks.

NOTE that I've used the COR values that Rebecca recently added to the WAS request. Let me know if you think we should change the Dropdown to an Input

After comments are received/implemented and I've merged, I'll likely be applying similar changes to the Terminal Access Request as well.

### CODE REVIEW

**Review by little-oddball — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        - Other - please specify in 'Additional Notes'
+    validations:
+      required: true
+  - type: checkboxes
+    attributes:
+      label: Please confirm if you are on the VFS Team Roster or Platform Team Roster
+      description: If you are on neither, close this request and reach out to your Product Manager to be added
+      options:
+        - label: "[VFS Team Roster](https://docs.google.com/spreadsheets/d/11dpCJjhs007uC6CWJI6djy3OAvjB8rHB65m0Yj8HXIw/edit?folder=0ALlyxurHpUilUk9PVA#gid=2042046665)"
+        - label: "[Platform Team Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster)"
```
Comment (rmtolmach) [inline .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml:72]: You don't want quotes on either of these `label`s.  I think that's why it's making the markdown look funky maybe? https://github.com/department-of-veterans-affairs/va.gov-team/blob/67bb24ff0c87d91ecc4070192865ed8f7cc147cf/.github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml

File: .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        - Delano McVay
+        - Jennifer O'Day
+        - Laurene "Reney" Cook
+        - Rolanda Copeland
+        - Other - please specify in 'Additional Notes'
+    validations:
+      required: true
+  - type: checkboxes
+    attributes:
+      label: Please confirm if you are on the VFS Team Roster or Platform Team Roster
```
Comment (rmtolmach) [inline .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml:68]: @little-oddball can VFS teams view that VFS roster? I'm not sure what the permissions are on that.

File: .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml (hunk 1 above)
Comment (rmtolmach) [reply .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml:72]: You _might_ get people saying that they can't view that Platform roster. It's confluence so VFS teams can't see it.

File: .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        - label: sandbox
+        - label: production (if requesting, justify prod access in 'Additional Notes' section)
+    validations:
+      required: true
+  - type: checkboxes
+    attributes:
+      label: If PRODUCTION ACCESS is required, confirm/complete ONE of the following
+      description: If you have not submitted your E-QIP questionaire, Production access will not be provided.
+      options:
+        - label: "I have an AWS Username and it is listed in [here](https://github.com/department-of-veterans-affairs/devops/blob/master/terraform/environments/global/iam_users.tf)"
```
Comment (rmtolmach) [inline .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml:91]: The quotes might be messing up this markdown formatting.

File: .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml (hunk 1 above)
Comment (ryan-mcneil) [reply .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml:72]: I **think** the quotes are fine. The preview is wonky, and it threw me off too, but that's how it is in our other forms.

But the Roster thing is a bummer. I gotta rethink this now, as that kinda screws up my plan here 😢

File: .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml (hunk 2 above)
Comment (ryan-mcneil) [reply .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml:68]: just confirmed that anyone with the link can view 👍

File: .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml (hunk 1 above)
Comment (ryan-mcneil) [reply .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml:72]: I think we're good here 👍

File: .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml (hunk 2 above)
Comment (rmtolmach) [reply .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml:68]: oh, nice!

File: .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml (hunk 1 above)
Comment (rmtolmach) [reply .github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml:72]: You're right! My bad! I just confirmed by looking at the AWS access template (preview markdown vs real open issue). Looks wonky by renders fine ✅  
image

**Review by rmtolmach — APPROVED** [review]

**Pull request merged**